badger/cmd/info.go

```diff
@@ -322,7 +322,7 @@ func printInfo(dir, valueDir string) error {
 			fp.Close()
 		}
 	}()
-	manifest, truncOffset, err := badger.ReplayManifestFile(fp)
+	manifest, truncOffset, err := badger.ReplayManifestFile(fp, 0)
```

Comment:
This should not be `zero`. We won't be able to run `badger info` on the directory with an external magic version set.